src/plugins/plugin.tooltip.js

```diff
@@ -275,7 +278,9 @@ function getTooltipSize(tooltip) {
 			+ options.titleMarginBottom;
 	}
 	if (combinedBodyLength) {
-		height += combinedBodyLength * bodyFontSize
+		// Body lines may include some extra height depending on boxHeight
+		const bodyLineHeight = bodyFontSize > boxHeight ? bodyFontSize : boxHeight;
+		height += combinedBodyLength * bodyLineHeight
```

Comment:
With the change in #6984, this isn't correct when `boxHeight > bodyFontSize`. I think the correct version is

```javascript
height += combinedBodyLength ? bodyLineHeight + ((combinedBodyLength - 1) * bodyFontSize) : 0;
```

Reply:
@etimberg Makes sense, you don't want to allocate all that space for the colorBox on lines where the colorbox is no longer drawn. Do you happen to have the configuration you were using for the chart in your screenshot handy?

Reply:
Not off hand. I believe I used https://github.com/chartjs/Chart.js/blob/master/samples/charts/bar/vertical.html and changed the labels on https://github.com/chartjs/Chart.js/blob/master/samples/charts/bar/vertical.html#L30 to

```javascript
['January', 'February', 'March', ['April', 'line 2'], 'May', 'June', 'July']
```